packages/cli/src/commands/build.ts

```diff
@@ -261,11 +261,15 @@ export async function build({
   // save the state to ipfs
   const miscUrl = await runtime.recordMisc();
 
+  const chainDef = def.toJson();
+
+  chainDef.version = chainDef.version.startsWith('<%=') ? pkgVersion : chainDef.version;
```

Comment:
I added this line to store chain definition version with interpolated value, before this when trying to publish a package with a interpolated version it had no way of knowing what that packages version is without it being in the chain definition